src/deluge/gui/views/automation_instrument_clip_view.cpp

```diff
@@ -1236,16 +1350,16 @@ ActionResult AutomationInstrumentClipView::padAction(int32_t x, int32_t y, int32
 	InstrumentClip* clip = getCurrentClip();
 	Instrument* instrument = (Instrument*)clip->output;
 
-	if (instrument->type == InstrumentType::CV) {
-		displayCVErrorMessage();
-		return ActionResult::DEALT_WITH;
-	}
-
 	char modelStackMemory[MODEL_STACK_MAX_SIZE];
 	ModelStackWithTimelineCounter* modelStack = currentSong->setupModelStackWithCurrentClip(modelStackMemory);
 
 	// Edit pad action...
 	if (x < kDisplayWidth) {
```

Comment:
moved display of this error message down so that mute and audition pad actions work in CV clip types, but regular edit pad actions (in the sequencer section of the grid, do not)